packages/terra-clinical-result/src/ClinicalResult.module.scss

```diff
@@ -0,0 +1,92 @@
+:local {
+  .decorated-result-display {
+    align-items: baseline;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    max-width: 100%;  // Needed for IE10 truncation
+    overflow: hidden; // Needed for IE10
+
+    &.status-in-error del {
+      color: var(--terra-clinical-result-display-status-entered-in-error-color, rgba(28, 31, 33, 0.65));
+      text-decoration: line-through;
+    }
+
+    &.truncated {
+      flex-wrap: nowrap;
+
+      .result-display {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        word-break: normal;
+        word-wrap: normal;
+      }
+    }
+
+    .result-display {
+      overflow: visible;
+    }
+
+    .icon-modified {
+      color: var(--terra-clinical-result-display-icon-modified-color, #26a2e5);
+      flex: 0 0 auto;
+      height: var(--terra-clinical-result-display-icon-modified-height, 0.875em); // Must use ems for font scaling
```

Comment:
Do each of these icons need their own height? (and should they be the same?) Could we create an icon class for the repeated styles?

```
.icon {
      flex: 0 0 auto;
      overflow: hidden;
      position: relative;
      vertical-align: baseline;
}

```

Reply:
they have the option to be uniquely sized and positioned

Reply:
+1 on creating a shared class for the other non-themeable items (flex, vertical-align, position, overflow)